draft-ietf-mops-streaming-opcons.md

```diff
@@ -553,10 +553,20 @@ Streaming media latency refers to the "glass-to-glass" time duration, which is t
 
 Streaming media can be usefully categorized according to the application's latency requirements into a few rough categories:
 
-- ultra low-latency    (less than 1 second)
-- low-latency live     (less than 10 seconds)
-- non-low-latency live (10 seconds to a few minutes)
-- on-demand            (hours or more)
+- near realtime latency (less than 150 milliseconds)
+- ultra low-latency     (less than 1 second)
+- low-latency live      (less than 10 seconds)
+- non-low-latency live  (10 seconds to a few minutes)
+- on-demand             (hours or more)
+
+## Near Realtime Latency
```

Comment:
My first instinct for the explanation section would be aiming for some text that hits the following points:

 - explaining that this latency category is driven by interactivity requirements
 - that a variety of interactive applications exist (and are envisioned) and they have a variety of different latency targets
 - that the latency requirements are driven by a broader system that includes the application's interactivity feedback mechanism and sensitivity to timing of the response
 - a full review of the use cases and techniques for satisfying them is out of scope for this doc, even though these applications sometimes do incorporate elements of streaming media with hard latency requirements.

(As long as we say something along these lines, I'm fine with also listing a few known example applications, especially if they have references for further reading.)

It also could be worth mentioning that some of the points in the next category (ultra low latency) are applicable here as well, or moving some of the text to this section, if we add it.

Reply:
I am fine with adding some text @GrumpyOldTroll after our current categories and just before Section 4.1, but I am *not* OK to add another category to the list named "near realtime latency (less than 200 milliseconds)" or by any other name to that effect. This is a can of worms, as soon as we out 200ms in there, someone else will start complaining about why not 50 ms. We are obviously not denying the need for such latencies but adding that category is not going to help anybody. 

New text should be short and crisp, something along the lines of:
We note that some emerging applications and services _(e.g., list a few examples if desired)_ may need latencies in the order of a few hundreds of milliseconds or even less driven by interactivity requirements. However, the specific use cases and techniques for satisfying these requirements is out of scope for this document.

Reply:
I agree we should not add a category.  I also agree it's a can of worms and the main point of this discussion is to close it better so that people stop asking about it when they read this bit.

I do think it would be useful to add a sub-section under ultra-low latency to break it away from the rest of the text there, since it's a bit long already.  I made a [new PR](https://github.com/ietf-wg-mops/draft-ietf-mops-streaming-opcons/compare/issue-130) from my [latest proposed text](https://github.com/ietf-wg-mops/draft-ietf-mops-streaming-opcons/pull/129#issuecomment-1110513354) in the long thread.  I was just aiming to do that separation and clarify the "Applications with further-specialized latency requirements" in our out-of-scope statement a little more specifically.

I don't mind changing it to your text if you like it better, but I thought putting it at the end in place of our existing "out of scope latency" statement was a smaller change.